handbook/community.md

```diff
@@ -0,0 +1,38 @@
+## Fleet community
+
+As an open-core company, Fleet endeavors build a community of engaged users, customers, and
+contributors.
+
+### Community contributions (pull requests)
+
+The top priority when community members contribute PRs is to help the person feel engaged with
```

Comment:
> This means acknowledging the contribution quickly (within 1 business day), and driving to a
resolution (close/merge).

@zwass FYI: the first time I read this, I thought it meant we will target closing/merging PRs within 1 business day  (which I think won't always be possible)